src/main/java/racingcar/racinggame/RacingGame.java

```diff
@@ -52,4 +56,14 @@ public boolean isEnd(){
     public int getPlayCount() {
         return this.playCount;
     }
+
+    public List<Car> getWinner(){
+        Optional<Car> largest = carList.stream()
+                .max(Comparator.comparing(Car::getPosition));
```

Comment:
Optional은 비싼 자원이기 때문에 무심결에 잘못사용할 경우 안티패턴이 될 수 있습니다.
해당 로직은 Car가 Comparable을 구현한다면 다음과 같이 쉽게 변경할 수 있을 것 같아요 :)

```suggestion
        Car winner = Collections.max(carList);
```

Optional에 대해선 다음 링크를 참고하시면 좋을 것 같아요!
http://homoefficio.github.io/2019/10/03/Java-Optional-%EB%B0%94%EB%A5%B4%EA%B2%8C-%EC%93%B0%EA%B8%B0/